eng/pipelines/common/platform-matrix.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         ${{ if eq(parameters.passPlatforms, true) }}:
           platforms: ${{ parameters.platforms }}
         helixQueueGroup: ${{ parameters.helixQueueGroup }}
-        managedTestBuildOsGroup: Linux
+        managedTestTargetOsGroup: Linux
```

Comment:
The changes in this file should be reverted. The `managedTestBuildOsGroup` refers to which OS we are building the CoreCLR managed tests on.

Reply:
It looks like also the changes under `eng/pipelines/coreclr` should also be reverted for the same reason.

Reply:
yes I reverted all of them :)